pkg/vsphere/infrastructure/task/tasks.go

```diff
@@ -363,7 +363,7 @@ func (t *tier1GatewayLocaleServiceTask) ListAll(ctx EnsurerContext, state *api.N
 
 func (t *tier1GatewayLocaleServiceTask) EnsureDeleted(ctx EnsurerContext, state *api.NSXTInfraState) (bool, error) {
 	if t.IsExternal(state) {
-		return true, nil
+		return false, nil
```

Comment:
Why?

Reply:
This was a minor inconsistency. The returned boolean flags if the resource has been deleted.